### PR TITLE
Google prediction connector improvements

### PR DESCRIPTION
**Pull request by elilmatha**

### CODE REVIEW

File: googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleprediction-config/component.xml [hunk 1]
```diff
@@ -1,4 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (kesavany) [inline googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleprediction-config/component.xml]: Add the proper year in license header. Fix in whole PR.

File: googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleprediction-trainedmodels/getAnalyzeTrainedModel.xml [hunk 2]
```diff
@@ -14,7 +32,7 @@
 		<call>
 			<endpoint>
 				<http method="get"
-					uri-template="{uri.var.apiUrl}/prediction/v1.6/projects/{uri.var.project}/trainedmodels/{uri.var.modelId}/analyze{uri.var.fields}" />
+					uri-template="legacy-encoding:{uri.var.apiUrl}/prediction/v1.6/projects/{uri.var.project}/trainedmodels/{uri.var.modelId}/analyze{uri.var.fields}" />
```
Comment (kesavany) [inline googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleprediction-trainedmodels/getAnalyzeTrainedModel.xml]: Format the code.

File: googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java [hunk 3]
```diff
@@ -456,9 +456,8 @@ public void testDeleteTrainedModelWithMandatoryParameters() throws Exception {
                                 + connectorProperties.getProperty("project") + "/trainedmodels/"
                                 + connectorProperties.getProperty("modelId"), "DELETE", apiRequestHeadersMap);
         
-        Assert.assertEquals(204, esbRestResponse.getHttpStatusCode());
-        Assert.assertEquals(404, apiRestResponse.getHttpStatusCode());
-        
+        Assert.assertEquals(apiRestResponse.getHttpStatusCode(), esbRestResponse.getHttpStatusCode());
+
```
Comment (Shakila) [inline googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java]: This empty line is not needed. Please remove it.

File: googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java [hunk 4]
```diff
@@ -483,9 +482,7 @@ public void testDeleteTrainedModelWithNegativeCase() throws Exception {
                         apiRequestHeadersMap);
         
         Assert.assertEquals(apiRestResponse.getHttpStatusCode(), esbRestResponse.getHttpStatusCode());
-        Assert.assertEquals(esbRestResponse.getBody().getJSONObject("error").getString("message"), apiRestResponse
-                .getBody().getJSONObject("error").getString("message"));
-        
+
```
Comment (Shakila) [inline googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java]: This empty line is not needed. Please remove it.

File: googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleprediction-config/component.xml (hunk 1 above)
Comment (elilmatha) [inline googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleprediction-config/component.xml]: Fixed in latest repo https://github.com/wso2-extensions/esb-connector-googleprediction/blob/master/src/main/resources/googleprediction-config/component.xml

File: googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleprediction-trainedmodels/getAnalyzeTrainedModel.xml (hunk 2 above)
Comment (elilmatha) [inline googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleprediction-trainedmodels/getAnalyzeTrainedModel.xml]: Fixed.
https://github.com/wso2-extensions/esb-connector-googleprediction/blob/master/src/main/resources/googleprediction-trainedmodels/getAnalyzeTrainedModel.xml

File: googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java (hunk 3 above)
Comment (elilmatha) [inline googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java]: Fixed in latest https://github.com/wso2-extensions/esb-connector-googleprediction/blob/master/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java

File: googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java (hunk 4 above)
Comment (elilmatha) [inline googleprediction/gp-connector/gp-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java]: Fixed in https://github.com/wso2-extensions/esb-connector-googleprediction/blob/master/src/test/java/org/wso2/carbon/connector/integration/test/googleprediction/GooglePredictionConnectorIntegrationTest.java

**Pull request merged**